scripts/base/build_glew.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -ex
+
+if [ ! -f $DOWNLOADS_DIR/glew-${GLEW_VERSION}.tgz ]; then
+    curl --location https://github.com/nigels-com/glew/releases/download/glew-${GLEW_VERSION}/glew-${GLEW_VERSION}.tgz -o $DOWNLOADS_DIR/glew-${GLEW_VERSION}.tgz
+fi
+
+tar xf $DOWNLOADS_DIR/glew-${GLEW_VERSION}.tgz
+
+cd glew-${GLEW_VERSION}/build
+cmake ./cmake -DCMAKE_INSTALL_PREFIX=${ASWF_INSTALL_PREFIX}
+make -j4
```

Comment:
You can use:
cmake --build . -j 4
for a slightly more portable parallel build (assuming CMake 3.12 or newer)

Reply:
Unfortunately VFX-2018 is using cmake-3.9.4 so I can't update to this syntax yet...